requirements.txt

```diff
@@ -0,0 +1,7 @@
+matplotlib
+numpy
+obspy
+pandas
+pyproj
+scipy
+scikit-fmm==2019.1.30
```

Comment:
Are our requirements actually more specific? (If so, is it helpful to list those specifics here?)

Reply:
Currently, no? If a future dependency breaks, it’s better not to be pinned to a specific version. This way we can address the issue (fix or pin) as it comes up. Will depend on what you find with obspy inventory, but I think this is fine.

Reply:
I mean in terms of minimum versions that we require. So pandas > 1.0.0 etc. For obspy it would be similar (haven't checked yet), something like obspy > 1.2.0.

Again, not certain that there actually are any other cases of this.

Reply:
I'm sure there is some level of requirement for pyproj, scipy, numpy and matplotlib. Whether it's worth digging into, or safe enough to rely on the obspy requirements to sort us out I don't know. They have:
`INSTALL_REQUIRES = [
    'numpy>=1.15.0',
    'scipy>=1.0.0',
    'matplotlib>=3.2.0',
    'lxml',
    'setuptools',
    'sqlalchemy',
    'decorator',
    'requests']`

I had a look back to when I had trouble with pyproj before, can't see any obvious breaking point, so I would leave that unless we have issues.